lib/phoenix_live_view.ex

```diff
@@ -1819,7 +1820,7 @@ defmodule Phoenix.LiveView do
         Enum.reduce(items, new_socket, fn item, acc -> stream_insert(acc, name, item, opts) end)
 
       %{} ->
-        config = get_in(streams, [:__configured__, name]) || []
+        config = get_in(streams, [:__configured__, name]) || opts
```

Comment:
We didn't pass the limit when initially creating a stream

Reply:
I think there might be another problem here when using `stream_configure`. In that case the opts are always ignored, right? Will need to add a test.

Probably something like `Keyword.merge(opts, get_in(streams, [:__configured__, name]) || [])`

Reply:
https://github.com/phoenixframework/phoenix_live_view/pull/3035/commits/d4bc58ac3d4bde26e1f89631348c4f3b4bc44103